adafruit_bitmap_font/bdf.py

```diff
@@ -105,15 +112,15 @@ def _verify_bounding_box(self):
                 "Source file does not have the FOUNTBOUNDINGBOX parameter"
             ) from error
 
-    def _readline_file(self):
+    def _readline_file(self) -> str:
         line = self.file.readline()
         return str(line, "utf-8")
 
-    def get_bounding_box(self):
+    def get_bounding_box(self) -> Tuple[int, int, int, int]:
         """Return the maximum glyph size as a 4-tuple of: width, height, x_offset, y_offset"""
         return self._boundingbox
 
-    def load_glyphs(self, code_points):
+    def load_glyphs(self, code_points: Union[int, str, set]) -> None:
```

Comment:
It looks like the 4th option is that `set()` will be used to cast the value into a `set`.  In that case, we could add `typing.Iterable` to the `Union`, but since `set` is an `Iterable`, it can just be:

```suggestion
    def load_glyphs(self, code_points: Union[int, str, Iterable]) -> None:
```

Don't forget to import `Iterable` from `typing` up above!